publications/2024_jaszowiec/poster.tex

```diff
@@ -0,0 +1,180 @@
+        Electric hotplate that can reach up to \textbf{400°C}. Based on the
+        firebricks and resistance wire. Used for soft and post-exposure bake of
+        photoresist, synthesis of tetraethyl orthosilicate based Spin-On-Dopant
+        and it's predeposition step before diffusion doping in the furnace.
+        Raspberry Pi Pico W microcontroller\cite{picow} is used for wireless
+        access, controlling rate of heating and automatic temperature steering.
+        We provide the microcontroller \textbf{firmware as an open source
+        software}\cite{firmware}.
+    }
+
+    \block{Doping}{
+        \textbf{We propose a modified process to the one described in Fangsuwannarak et
+        Al. 2019 \cite{Fangsuwannarak}}. It yields a less, but sufficiently viscous
+        dopant and significantly \textbf{extends it's shelf life in room temperature}.
+        We spin coat the dopant on the wafer, do pre deposition on hot plate
+        and thermal diffusion in the furnace. This creates
+        N-doped wells and PSG. The PSG is stripped using \( NH_{4}F\cdot HF\)
+        solution dissolved in DI water.
+        For \textbf{selective doping} we use thick (\(\sim\)500nm) thermally grown
+        \(SiO_{2}\), that stops spin coated SOD from penetrating too far into
+        the wafer.
+        We control for rate of doping by doing a \textbf{double-oxidation}
+        process, which grows two layers of \(SiO_{2}\), one, for completely
+        stopping the doping \(\sim\)500nm and another in a range of 30-100nm,
+        which acts as a doping limiter and not a complete stopper.
+    }
+
+    \block{Spin coating}{
+        We successfully used a 3000 RPM, 12V \textbf{computer fan} to
+        manufacture solar cells and NPN MOSFET devices with a double-sided tape
+        on which the wafer is placed. The solution, while very cheap and
+        simple, but is not very convenient. The tape has to be replaced
+        frequently and contaminates the back of the wafer. For this reason, we
+        \textbf{designed and 3D-printed} a \textbf{rotating bed} with an
+        opening in the middle and a chamber.  The bed is connected to the
+        \textbf{BL DC motor} that can spin the bed to up to \textbf{4000 RPM}.
+        We use \textbf{infrared light source and detector}, connected to the
+        Raspberry Pi Pico microcontroller\cite{picow}, forming a
+        \textbf{feedback} and allowing us to \textbf{control the rotating speed
+        of the bed}. The opening in the bed is connected through a hose to the
+        modified \textbf{vacuum cleaner} that sucks the wafer down to the bed.
+    }
+
+    \block{References}{
+        \nocite{*}
+        \printbibliography[heading=none]
+    }
+\end{columns}
+
+\begin{columns}
+    \column{1.00}
+    \block{Testing \& results}{
+        We were able to make various devices, including \textbf{resistors},
+        \textbf{solar cells}, \textbf{capacitors} and \textbf{MOSFETs}. \\
```

Comment:
Should we mention diodes separately from solar cells ?

Reply:
We can't really call it diodes, I think. Reverse bias effect is just too weak, so let's just stick with solar cells, because they do work.